conan/tools/cmake/cmakedeps.py

```diff
@@ -33,18 +33,18 @@
 
 
 conan_package_library_targets = textwrap.dedent("""
-   function(conan_package_library_targets libraries package_libdir deps out_libraries out_libraries_target build_type package_name)
+   function(conan_package_library_targets libraries package_libdir deps out_libraries out_libraries_target config_suffix package_name)
        unset(_CONAN_ACTUAL_TARGETS CACHE)
 
        foreach(_LIBRARY_NAME ${libraries})
            find_library(CONAN_FOUND_LIBRARY NAME ${_LIBRARY_NAME} PATHS ${package_libdir}
                         NO_DEFAULT_PATH NO_CMAKE_FIND_ROOT_PATH)
            if(CONAN_FOUND_LIBRARY)
-               conan_message(STATUS "Library ${_LIBRARY_NAME} found ${CONAN_FOUND_LIBRARY}")
+               conan_message(DEBUG "Library ${_LIBRARY_NAME} found ${CONAN_FOUND_LIBRARY}")
```

Comment:
Might be a good idea to keep the ".a" found as DEBUG because we have the new traces with the targets that probably is more useful.